tests/console/libjpeg_turbo.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: libjpeg_turbo
+# Summary: Check libjpeg_turbo version
+# - Test should test on sle15sp6, see https://jira.suse.com/browse/PED-4889
+# - Check libjpeg-turbo version is higher than 2.1.1
+# - Do functional check with jpegtran command
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use version_utils;
+use utils qw(zypper_call upload_y2logs);
+use registration qw(add_suseconnect_product is_phub_ready);
+
+sub run {
+    select_serial_terminal;
+
+    # Package 'libjpeg-turbo' requires PackageHub is available
+    add_suseconnect_product('PackageHub') unless is_phub_ready();
+
+    # Install libjpeg-turbo package
+    zypper_call("install libjpeg-turbo");
+    my $install_version = script_output('rpm -q libjpeg-turbo --qf %{version}');
+    record_info("libjpeg-turbo version", $install_version);
+    die("libjpeg-turbo version was not updated") if (package_version_cmp($install_version, '2.1.1') <= 0);
+
+    # Use jpegtran command to rotate image
+    zypper_call("install ImageMagick");
+    my $image = 'lizard.jpeg';
+    assert_script_run "wget --quiet " . data_url("imagemagick/$image");
+    assert_script_run("chmod +x $image");
```

Comment:
Why?

Reply:
I need upload a new image with the height and weight have different value. Like: 279x180
Then do rotation 90 degree, it changed to 176x279
As the command uses '-trim' it will discard unrotatable edge pixels
To  install 'ImageMagick',  because I need use 'identify' command to get the image size.

Reply:
None of that should require executable permissions on the image file